### PR TITLE
add two more global reports

### PR DESCRIPTION
**Pull request by FuhuXia**
For https://github.com/GSA/data.gov/issues/4943
## Added two global reports:
- MOST_DOWNLOADED_DATASETS
- MOST_CLICKED_OUTBOUND_LINKS

## security considerations
[Note the any security considerations here, or make note of why there are none]

### CODE REVIEW

**Comment by tdlowden** [comment]
@FuhuXia i made some slight ordering changes and removed a couple words from descriptions. I meant to suggest it but accidentally committed to the branch, Can you just give 👀 on my changes before we get a merge approval?

**Comment by tdlowden** [comment]
Otherwise, this is awesome,  ty!

**Review by jbrown-xentity — APPROVED** [review]

**Review by btylerburton — APPROVED** [review]

**Pull request merged**
